understanding/20/use-of-color.html

```diff
@@ -63,6 +63,17 @@ <h2>Intent of Use of Color</h2>
 
       </div>
 
+     <div class="note">
+       
+       <p>This criterion does not apply to situations where color has <em>not</em> been used to convey information, indicating an action,
+         prompting a response, or distinguishing a visual element. For instance, a hyperlink which has been styled to appear no
```

Comment:
Text looks good to me! Grammar nIt:

```suggestion
       <p>This criterion does not apply to situations where color has <em>not</em> been used to convey information, indicate an action,
         prompt a response, or distinguish a visual element. For instance, a hyperlink which has been styled to appear no
```